### PR TITLE
New module, :app everywhere

### PR DESCRIPTION
**Pull request by tecosaur**
No need to leave Emacs any more.

Module discussed on discord. See the `README.org` for more information on this.

### CODE REVIEW

File: modules/app/everywhere/config.el [hunk 1]
```diff
@@ -0,0 +1,29 @@
+;;; app/everywhere/config.el -*- lexical-binding: t; -*-
+
+(use-package! emacs-everywhere
+  :config
```
Comment (ymarco) [inline modules/app/everywhere/config.el:4]: This would eager-load emacs-everywhere

Comment (tecosaur) [reply modules/app/everywhere/config.el:4]: Hmmm, `:defer` would probably be a good idea wouldn't it.

Comment (tecosaur) [reply modules/app/everywhere/config.el]: Added.

**Review by hlissner — APPROVED** [review]

**Comment by hlissner** [comment]
Great work! Thanks again!

**Pull request merged**